gtk4/src/subclass/widget.rs

```diff
@@ -1,21 +1,132 @@
 use gtk_sys;
 
+use libc::c_int;
```

Comment:
Why not use `u32` instead?

Reply:
because the parent implementation requires a pointer

Reply:
So? `*mut i32` is the same as `*mut libc::c_int` for all platforms that glib supports